assets/prometheus-k8s/prometheus-rule.yaml

```diff
@@ -0,0 +1,262 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    app.kubernetes.io/component: prometheus
+    app.kubernetes.io/managed-by: cluster-monitoring-operator
+    app.kubernetes.io/name: prometheus
+    app.kubernetes.io/part-of: openshift-monitoring
+    app.kubernetes.io/version: 2.24.0
+    prometheus: k8s
+    role: alert-rules
+  name: k8s-rules
```

Comment:
same here about the name.

Reply:
I fixed this in kube-prometheus as this is where it came from - https://github.com/prometheus-operator/kube-prometheus/pull/940